virtio-devices/src/vdpa.rs

```diff
@@ -97,8 +123,29 @@ impl Vdpa {
         device_path: &str,
         mem: GuestMemoryAtomic<GuestMemoryMmap>,
         num_queues: u16,
+        restoring: bool,
     ) -> Result<Self> {
-        let mut vhost = VhostKernVdpa::new(device_path, mem).map_err(Error::CreateVhostVdpa)?;
+        if restoring {
+            return Ok(Vdpa {
+                common: VirtioCommon {
+                    queue_sizes: vec![1024; num_queues as usize],
```

Comment:
Why pick the default queue size as 1024? If it is a placeholder, wouldn't it be better to use a value that's apparently invalid (e.g. 0)?

Reply:
I know it's not ideal, but that's a direct consequence of the fact that we can't restore the `Vdpa` device before the associated `VirtioPciDevice` gets created. We need a somehow valid `queues_sizes` here, which means I chose 1024 so that it's big enough.